### ISSUE TITLE
Chrome apps

### ISSUE THREAD

**Issue by captainwasabi** [tjackenpacken/taskbar-groups#237]
You can't add them to a group.  it throws an exception.

Unfortunately, I have too many chrome apps in my workflow to use this. 

Doug

**Comment by ghost** [comment]
Yeah, I have a problem with chrome apps too. They won't open but no error or anything.

**Comment by pbriquet** [comment]
Same problem here... No error but not opening.
